lib/features/thread_detail/presentation/extension/thread_detail_on_selected_email_updated.dart

```diff
@@ -19,15 +20,35 @@ extension ThreadDetailOnSelectedEmailUpdated on ThreadDetailController {
       return;
     }
 
-    if (currentExpandedEmailId.value != null) {
-      mailboxDashBoardController.dispatchEmailUIAction(
-        DisposePreviousExpandedEmailAction(
-          currentExpandedEmailId.value!,
-        ),
-      );
+    scrollController = ScrollController();
```

Comment:
IMO, We should only create it when it is not created yet, `scrollController == null` or scrollController is disposed